lib/taskana-core/src/main/java/pro/taskana/spi/history/api/events/task/CancelledEvent.java

```diff
@@ -0,0 +1,13 @@
+package pro.taskana.spi.history.api.events.task;
+
+import pro.taskana.task.api.models.Task;
+
+/** Event fired if a task is cancelled. */
+public class CancelledEvent extends TaskEvent {
+
+  public CancelledEvent(String id, Task task, String userId) {
+    super(id, task, userId, null);
+    eventType = "TASK_CANCELLED";
+    created = task.getModified();
```

Comment:
correct would be to use the completed timestamp, which should be identical.

Reply:
The timestamp is identical, but I will change it to task.getCompleted()

Reply:
You will but you haven't done it yet, correct? Or is this just an displaying issue?

Reply:
I ave this changed but not pushed yet, since the chances in the asserts are not yet agreed upon. I'm going to push everything together